### ISSUE TITLE
another gluon implementation 

### ISSUE THREAD

**Issue by Cjiangbpcs** [haofanwang/mxnet-headpose#2]
Hi, I also wrote a gluon version of the original pytorch model. Unlike yours, I used gluon model zoo pretrained model, the github is here: https://github.com/Cjiangbpcs/gazenet_mxJiang

There are some issues though:
1. my model does not run as fast as the original model;
2. my model does not converge the same way as the original model;

For 1, my guess is that I separated the model to 3 separate model files. It reloads the pretrianed model every time the models are run, that might cause slowdown of the training.

For 2, I have changed the pretrained model batchnorm training mode to evaluation mode by switching the use_global_states = True, to reproduce the pytorch model.eval() functionality during the training process. 

From your experience of training your version, did you see problems in my implementation/code? Can you share your thoughts about why problem 1 and 2?

Thank you.

**Comment by Cjiangbpcs** [comment]
just an update, my gluon re-implementation works fine and converges as fast as or even slightly faster than the Pytorch version. Close this issue.
